lib/pages/movie_info_screen/app_movie_info.dart

```diff
@@ -0,0 +1,44 @@
+import 'package:flutter/material.dart';
+import 'package:the_movie_db_module2_part1/model/movie_data_model.dart';
+import 'package:the_movie_db_module2_part1/pages/movie_info_screen/genres_list_widget.dart';
+import '../../app_header.dart';
+import 'related_genres_title.dart';
+import 'movie_overview.dart';
+import 'movie_info_container.dart';
+
+class AppMovieInfo extends StatefulWidget {
+  const AppMovieInfo({super.key});
+
+  @override
+  State<AppMovieInfo> createState() => _AppMovieInfoState();
+}
+
+class _AppMovieInfoState extends State<AppMovieInfo> {
+  static const double sizedBoxHeight = 10;
+
+  @override
+  Widget build(BuildContext context) {
+    final MovieDataModel movie = ModalRoute.of(context)?.settings.arguments as MovieDataModel;
+    String backdropImagePath = 'https://image.tmdb.org/t/p/original/${movie.backdropPath}';
```

Comment:
This could be moved to a custom getter inside `MovieDataModel`, like `posterPath`